tests/pytests/unit/modules/test_syslog_ng.py

```diff
@@ -0,0 +1,363 @@
+"""
+    Test cases for salt.modules.syslog_ng
+"""
+
+
+import os
+from textwrap import dedent
+
+import pytest
+
+import salt.modules.syslog_ng as syslog_ng
+from tests.support.mock import MagicMock, patch
+
+
+@pytest.fixture
+def _VERSION():
+    return "3.6.0alpha0"
+
+
+@pytest.fixture
+def _MODULES():
+    return (
+        "syslogformat,json-plugin,basicfuncs,afstomp,afsocket,cryptofuncs,"
+        "afmongodb,dbparser,system-source,affile,pseudofile,afamqp,"
+        "afsocket-notls,csvparser,linux-kmsg-format,afuser,confgen,afprog"
+    )
+
+
+@pytest.fixture
+def VERSION_OUTPUT(_VERSION, _MODULES):
+    return """syslog-ng {0}
+Installer-Version: {0}
+Revision:
+Compile-Date: Apr  4 2014 20:26:18
+Error opening plugin module; module='afsocket-tls', error='/home/tibi/install/syslog-ng/lib/syslog-ng/libafsocket-tls.so: undefined symbol: tls_context_setup_session'
+Available-Modules: {1}
+Enable-Debug: on
+Enable-GProf: off
+Enable-Memtrace: off
+Enable-IPv6: on
+Enable-Spoof-Source: off
+Enable-TCP-Wrapper: off
+Enable-Linux-Caps: off""".format(
+        _VERSION, _MODULES
+    )
+
+
+@pytest.fixture
+def STATS_OUTPUT():
+    return """SourceName;SourceId;SourceInstance;State;Type;Number
+center;;received;a;processed;0
+destination;#anon-destination0;;a;processed;0
+destination;#anon-destination1;;a;processed;0
+source;s_gsoc2014;;a;processed;0
+center;;queued;a;processed;0
+global;payload_reallocs;;a;processed;0
+global;sdata_updates;;a;processed;0
+global;msg_clones;;a;processed;0"""
+
+
+@pytest.fixture
+def orig_env():
+    return {"PATH": "/foo:/bar"}
+
+
+@pytest.fixture
+def bin_dir():
+    return "/baz"
+
+
+@pytest.fixture
+def mocked_env():
+    return {"PATH": "/foo:/bar:/baz"}
+
+
+@pytest.fixture
+def configure_loader_modules():
+    return {syslog_ng: {}}
+
+
+def test_statement_without_options():
+    s = syslog_ng.Statement("source", "s_local", options=[])
+    b = s.build()
+    assert (
```

Comment:
```suggestion
    assert b == (
```
Always assert what we get with what we expect.
In case of failures, if a diff can be made from the object by pytest, it will make more sense.

There are several assertions like this one.